bin/elm-petstore-all.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+./bin/elm-0.18-petstore.sh
```

Comment:
I think we should also add elm-format --validate to the CI as well to validate it was run.

Reply:
Sure. I'll do that in another PR by updating the CI config

Reply:
OK, great